src/components/Calculator.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+function Calculator() {
```

Comment:
- [ ] Based on the requirements of this project, the Calculator component should have at least  one child component. Passing data from a parent component to a child component in React is a crucial aspect in building powerful and dynamic user interfaces. You can refactor the codes into another component and pass it into the Calculator component :+1: